cryptolib/cryptolib.go

```diff
@@ -3,20 +3,28 @@ package cryptolib
 import (
 	"encoding/json"
 	"io/ioutil"
+	"log"
 	"net/http"
 	"strconv"
-	"strings"
+	"sync"
+	"time"
 
 	"github.com/JoeParrinello/brokerbot/messagelib"
 )
 
+var (
+	mu          sync.Mutex
+	priceFeeds  []*PriceFeed
+	lastUpdated time.Time
+)
+
 const (
 	geminiBaseURL      = "https://api.gemini.com"
 	geminiPriceFeedURI = "/v1/pricefeed"
 	brokerbotUserAgent = "brokerbot"
-)
 
-type contextKey string
+	priceFeedAgeLimit = 5 * time.Minute
```

Comment:
Should this be something that we have configurable with a command line flag? (defaulting to 5 minutes, but maybe for testing we want to keep it shorter or longer to debug)

Reply:
Switched to a flag.